requirements.txt

```diff
@@ -1 +1,3 @@
 pytest==3.1.1
+coverage==4.4.1
```

Comment:
pip installs `coverage` automatically when it installs `pytest-cov`. Please remove `coverage` from the requirements.txt.